public/users.json

```diff
@@ -0,0 +1,1000 @@
+[{"_id":"82be84f4-f04b-469c-8dba-abd150d6da1c","firstName":"Andra","lastName":"Heavens","email":"aheavens0@omniture.com","gender":"Female","isCoach":true,"isArchived":false,"isAdmin":false,"lang":"Danish","createdAt":"2020-09-07T12:13:12Z","updatedAt":"2021-01-09T06:02:50Z","phone":"9977639389"},
```

Comment:
This look like a mock data file, this shouldn't be here except if it's need for testing. I recommend you to add it inside the `.gitignore` as well